jetty-ee10/jetty-ee10-tests/jetty-ee10-test-client-transports/src/test/java/org/eclipse/jetty/ee10/test/client/transport/ServerTimeoutsTest.java

```diff
@@ -42,74 +42,61 @@
 import org.eclipse.jetty.http2.FlowControlStrategy;
 import org.eclipse.jetty.http2.client.transport.HttpClientTransportOverHTTP2;
 import org.eclipse.jetty.logging.StacklessLogging;
-import org.eclipse.jetty.server.Handler;
-import org.eclipse.jetty.server.Request;
-import org.eclipse.jetty.server.handler.AbstractHandler;
 import org.eclipse.jetty.server.internal.HttpChannelState;
 import org.eclipse.jetty.util.Callback;
 import org.eclipse.jetty.util.IO;
-import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.junit.jupiter.params.provider.MethodSource;
 
-import static org.eclipse.jetty.ee9.http.client.Transport.FCGI;
-import static org.eclipse.jetty.ee9.http.client.Transport.H2;
-import static org.eclipse.jetty.ee9.http.client.Transport.H2C;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.containsString;
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-
-public class ServerTimeoutsTest extends AbstractTest<TransportScenario>
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+// TODO: the server-side timeout mechanism should be completely reworked.
+//  Currently, HttpChannel.onFailure() is called, but timeouts are different
+//  since they may be ignored, so we don't want to remember errors if they are ignored.
+//  However, this behavior is historically so because of Servlets, and we
+//  may decide differently for Handlers.
+@Disabled
```

Comment:
Reads like a test assumption mistake?
But also that we need a version of this test fore core/handlers as well.

Reply:
The test assumption is correct, since this is a Servlet-based test.
I added a test class for core.